runtime/catalog-karapace/src/main/java/io/aklivity/zilla/runtime/catalog/karapace/internal/KarapaceEventFormatter.java

```diff
@@ -58,33 +51,44 @@ public String format(
         case UNRETRIEVABLE_SCHEMA_SUBJECT_VERSION:
         {
             KarapaceUnretrievableSchemaSubjectVersionExFW ex = extension.unretrievableSchemaSubjectVersion();
-            result = String.format(UNRETRIEVABLE_SCHEMA_SUBJECT_VERSION, asString(ex.subject()), asString(ex.version()));
+            result = String.format(
+                    "Unable to fetch schema for subject %s with version %s.",
+                    asString(ex.subject()),
+                    asString(ex.version())
+            );
             break;
         }
         case UNRETRIEVABLE_SCHEMA_SUBJECT_VERSION_STALE_SCHEMA:
         {
             KarapaceUnretrievableSchemaSubjectVersionStaleSchemaExFW ex = extension
                 .unretrievableSchemaSubjectVersionStaleSchema();
-            result = String.format(UNRETRIEVABLE_SCHEMA_SUBJECT_VERSION_STALE_SCHEMA, asString(ex.subject()),
-                asString(ex.version()), ex.schemaId());
+            result = String.format(
+                    "Unable to fetch schema for subject %s with version %s; using stale schema with id %d.",
+                    asString(ex.subject()),
+                    asString(ex.version()),
+                    ex.schemaId()
+            );
             break;
         }
         case UNRETRIEVABLE_SCHEMA_ID:
         {
             KarapaceUnretrievableSchemaIdExFW ex = extension.unretrievableSchemaId();
-            result = String.format(UNRETRIEVABLE_SCHEMA_ID, ex.schemaId());
+            result = String.format("Unable to fetch schema id %d.", ex.schemaId());
             break;
         }
         case RETRIEVED_SCHEMA_SUBJECT_VERSION:
         {
-            KarapaceRetrievableSchemaSubjectVersionExFW ex = extension.retrievableSchemaSubjectVersion();
-            result = String.format(RETRIEVED_SCHEMA_SUBJECT_VERSION, asString(ex.subject()), asString(ex.version()));
+            KarapaceRetrievableSchemaSubjectVersionExFW ex = extension.retrievedSchemaSubjectVersion();
```

Comment:
Looks like we need to rename `KarapaceRetrievableSchemaSubjectVersionExFW ` to `KarapaceRetrievedSchemaSubjectVersionExFW ` to align with naming conventions, agree?